src/adapter/src/notice.rs

```diff
@@ -328,6 +335,16 @@ impl fmt::Display for AdapterNotice {
             AdapterNotice::UnknownSessionDatabase(name) => {
                 write!(f, "session database {} does not exist", name.quoted())
             }
+            AdapterNotice::WebhookSourceCreated {
+                database,
+                schema,
+                name,
+            } => {
+                write!(
+                    f,
+                    "URL to POST data is 'https://<HOST>/api/webhook/{database}/{schema}/{name}'"
```

Comment:
These are gonna need some escaping to make them URL-safe!

Reply:
Ahhh, great call! I updated the code to make the database, schema, and item name URL safe. I also added a Rust test that directly takes the URL and POSTs to it, asserting success.